lib/core/src/persist/mod.rs

```diff
@@ -99,14 +103,30 @@ impl Persister {
         VALUES (?, ?, ?, ?, ?, ?)
         ",
             (
-                ptx.tx_id,
+                &ptx.tx_id,
                 ptx.timestamp,
                 ptx.amount_sat,
                 ptx.fees_sat,
                 ptx.payment_type,
                 ptx.is_confirmed,
             ),
         )?;
+        if let Some(PaymentDetails::Liquid {
+            destination,
+            description,
+        }) = details
+        {
+            tx.execute(
+                "INSERT OR REPLACE INTO payment_details (
```

Comment:
If the `payment_details` table only contains payment details for direct Liquid txs, shouldn't it be given a clearer name? Like `direct_tx_details` , `direct_onchain_tx_details` or similar?

Reply:
I see, makes sense.